src/routers/mod.rs

```diff
@@ -1,5 +1,28 @@
+use anyhow::Result;
+use pyo3::{Py, PyAny};
+
 pub mod const_router;
 pub mod middleware_router;
 pub mod router;
 pub mod types;
 pub mod web_socket_router;
+
+pub struct RouteType<T>(pub T);
```

Comment:
what does the `(pub T)` mean here??

Reply:
I used this struct as a wrapper for the `Method` and `MiddlewareRoute` types. RouteType is a struct that can contain any other type. `pub T` is an unnamed field of this struct, which is public and of type `T`. It can be accessed like this in an instance of type `RouteType`: `route_type.0`.

However now that I think about it it is probably unnecessary here, and I could just use a generic type instead of wrapping the parameter in this struct. I will push a new commit removing it soon

Reply:
Ah, that makes sense. Thank you for explaining 😄